man/man5/zfs-module-parameters.5

```diff
@@ -374,6 +370,19 @@ angular velocity disk drive.
 Use \fB1\fR for yes (default) and \fB0\fR for no.
 .RE
 
+.sp
+.ne 2
+.na
+\fBsend_holes_without_birth_time\fR (int)
+.ad
+.RS 12n
+When set, the hole_birth optimization will not be used, and all holes will
+always be sent on zfs send. Useful if you suspect your datasets are affected
```

Comment:
This is useful if....

Reply:
This was not my text. My change only moved this existing in the file. I have applied your change as part of this PR.